src/openzeppelin/tests/test_ownable.cairo

```diff
@@ -0,0 +1,88 @@
+use openzeppelin::access::ownable::Ownable;
+
+use starknet::{ContractAddress, contract_address_const, testing::set_caller_address};
```

Comment:
Even when in this case is just a function, we would be using `testing` module members many times, then following Rust best practices, I think it will be better for clarity to keep the convention of importing the module instead of the functions directly (the multiline imports could be another convention, with the exception of particularly related members, like `add` and a `mul` in an `operations` module)

```suggestion
use starknet::ContractAddress;
use starknet::contract_address_const;
use starknet::testing;
```

Reply:
Sounds good to me